requirements.txt

```diff
@@ -0,0 +1,45 @@
+boto==2.38.0
+Chameleon==2.22
+cryptacular==1.4.1
+ecdsa==0.13
+enum34==1.0.4
+Fabric==1.10.2
+glob2==0.4.1
+gunicorn==19.3.0
+ipython==3.2.1
+Jinja2==2.7.3
+Mako==1.0.1
+MarkupSafe==0.23
+meld3==1.0.2
+paramiko==1.15.2
+parse==1.6.6
+parse-type==0.3.4
+PasteDeploy==1.5.2
+pbkdf2==1.3
+psycopg2==2.6.1
+py==1.4.30
+pycparser==2.14
+pycrypto==2.6.1
+Pygments==2.0.2
+pyramid==1.5.7
+pyramid-chameleon==0.3
+pyramid-debugtoolbar==2.4
+pyramid-jinja2==2.5
+pyramid-mako==1.0.2
+pyramid-tm==0.12
+pytest==2.7.2
+pytest-bdd==2.14.1
+repoze.lru==0.6
+selenium==2.46.0
+six==1.9.0
+splinter==0.7.2
+SQLAlchemy==1.0.6
+-e git+http://github.com/meslater1030/taste-buddies@db45e96105f405ab09b113faa7dac92bcb1cead6#egg=tastebuddies-dev
+transaction==1.4.4
+translationstring==1.3
+venusian==1.0
+waitress==0.8.9
+WebOb==1.4.1
+zope.deprecation==4.1.2
+zope.interface==4.1.2
+zope.sqlalchemy==0.7.6
```

Comment:
this file probably needs a bit of cleanup.  For example, I doubt you are using all three template systems mentioned in here (chameleon, jinja2 and mako).